CHANGELOG.md

```diff
@@ -3,16 +3,42 @@
 All notable changes to this project will be documented in this file. This
 project adheres to [Semantic Versioning](http://semver.org/).
 
-## Unreleased
+## 0.63.0
 
-### `opa exec` now supports a timeout flag
+This release contains a mix of features, performance improvements, and bugfixes.
 
-Previously, `opa exec` could end up in situations [where it might hang forever](https://github.com/open-policy-agent/opa/issues/6613).
-This can be very undesirable behavior in build pipelines and CI systems!
+### Runtime, Tooling, SDK
+
+- cmd/exec: Add `--timeout` flag to `opa exec` to prevent infinite hangs. (#6633) ([#6613](https://github.com/open-policy-agent/opa/issues/6613)) authored by @philipaconrad
```

Comment:
Nit:  we can remove `(#6633)` or add a link. Didn't seem to render.

Reply:
ah, thanks for noticing. `#6613` is the one we want.